README.md

```diff
@@ -256,8 +256,8 @@ questions and get our support.
 [ci-url]: https://github.com/opencv/cvat/actions
 [gitter-img]: https://img.shields.io/gitter/room/opencv-cvat/public?style=flat
 [gitter-url]: https://gitter.im/opencv-cvat
-[coverage-img]: https://coveralls.io/repos/github/cvat-ai/cvat/badge.svg?branch=develop
-[coverage-url]: https://coveralls.io/github/cvat-ai/cvat?branch=develop
+[coverage-img]: https://codecov.io/github/opencv/cvat/branch/develop/graph/badge.svg?token=SC0BwtMJXv
```

Comment:
Why do we need token here?

Reply:
I took it from [here](https://app.codecov.io/gh/opencv/cvat/settings/badge). But you're right. We don't need it, it's probably for a private repo.